docs/api/diagram_editor/editor/config/connectionpoints_property.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: connectionPoints!!
+title: connectionPoints Property of Editor
+description: You can learn about the connectionPoints property of editor in the documentation of the DHTMLX JavaScript Diagram library. Browse developer guides and API reference, try out code examples and live demos, and download a free 30-day evaluation version of DHTMLX Diagram.
+---
+
+# connectionPoints
+
+### Description
+
+@short: Optional. Enables/disables an ability to connect shapes using connection controls
+
+:::info
+The `connectionPoints` property works only if `type: "default"`.
```

Comment:
Suggestion: The `connectionPoints` property works only if the 'type:default' configuration option is specified.